internal/graph/extensions/live_query/resolve_live_query.gotpl

```diff
@@ -0,0 +1,37 @@
+{{ reserveImport "context" }}
+{{ reserveImport "time" }}
+{{ reserveImport "github.com/99designs/gqlgen/graphql" }}
+{{ reserveImport "github.com/upbound/xgql/internal/graph/extensions/live_query/runtime" }}
+
+func (ec *executionContext) __resolve_liveQuery(ctx context.Context, throttle *int) (<-chan graphql.Marshaler, error) {
```

Comment:
Why do we need to dynamically create this function?

Reply:
this resolver is injected into generated code. the only way I found to do it is by [marking the field as a resolver during code generation](https://github.com/upbound/xgql/pull/123/files#diff-d2dea1d50cfdef41ec0578a1f87ed83078985789341b7e27cb7d14efdeca271bR82-R109).